assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/data/useInitialState.ts

```diff
@@ -27,9 +38,17 @@ const useInitialState = ({ ticketId }: BaseProps): StateInitializer => {
 	//sort'em
 	const sortedPrices = sortByPriceOrderIdAsc(unSortedPrices);
 
+	const getPrices = () => {
+		const hasBasePrice = sortedPrices.filter(isBasePrice).length;
+
+		if (hasBasePrice) return sortedPrices;
+
+		return [basePrice];
```

Comment:
Safe to add `sortedPrices` to the array
```suggestion
		return [basePrice, ...sortedPrices];
```